Makefile

```diff
@@ -42,3 +42,50 @@ build:
 .PHONY: clean
 clean:
 	rm -rf integration/testdata/fixtures/
+
+$(GOBIN)/bbolt:
+	go get -v github.com/etcd-io/bbolt/cmd/bbolt
+
+export DB_TYPE ?= trivy
+
+ifeq ($(DB_TYPE),trivy-light)
+	DB_ARG := --light
+endif
+
+trivy-db:
+	make build
+
+.PHONY: db-all
+db-all:
+	make build db-fetch
+	make db-build
+	make db-compact
+	make db-compress
+
+.PHONY: db-fetch
+db-fetch:
```

Comment:
Now, even when security advisories are already under cache directory, `db-fetch` fetches all advisories. Is it possible to avoid it?

Reply:
Initially I wanted to avoid that we have a stale cache directory, so downloading them all seems to be the better option.

I can create a non-phony target for cache/ at the risk of not downloading them at all or having a stale fetch.

Reply:
Fair enough. We can keep it as is.